core/src/main/scala/org/apache/spark/ExecutorAllocationManager.scala

```diff
@@ -185,6 +195,12 @@ private[spark] class ExecutorAllocationManager(
    * If not, throw an appropriate exception.
    */
   private def validateSettings(): Unit = {
+
+    if ( streamingDRAFeatureEnabled && !Utils.isDynamicAllocationEnabled(conf) ) {
```

Comment:
nits, there are a bunch of formatting issues here, with extra spaces, and indentation and things wrong, please fix those

Reply:
I just realized that we are doing Utils.isDynamicAllocationEnabled(conf) check before creating ExecutorAllocationManager object at [spark context](https://github.com/apache/spark/blob/master/core/src/main/scala/org/apache/spark/SparkContext.scala#L663). I will remove this check